test/libsolidity/semanticTests/storage/packed_storage_signed.sol

```diff
@@ -13,14 +13,14 @@ contract C {
             b = (0 - uint8(a)) * 2;
             c = a * int8(120) * int8(121);
         }
-        x1 = uint256(a);
+        x1 = uint256(uint8(a));
```

Comment:
```suggestion
        x1 = uint256(uint256(a));
```

Reply:
That is not allowed since we can't convert from `int8` to `uint256`.